modules/profile/files/jenkins/agent_gpu/etc/X11/xorg.conf

```diff
@@ -0,0 +1,52 @@
+Section "ServerLayout"
+    Identifier     "Layout0"
+    InputDevice    "Keyboard0" "CoreKeyboard"
+    InputDevice    "Mouse0" "CorePointer"
+EndSection
+
+Section "Files"
+EndSection
+
+Section "InputDevice"
+    # generated from default
+    Identifier     "Mouse0"
+    Driver         "mouse"
+    Option         "Protocol" "auto"
+    Option         "Device" "/dev/psaux"
+    Option         "Emulate3Buttons" "no"
+    Option         "ZAxisMapping" "4 5"
+EndSection
+
+Section "InputDevice"
+    # generated from default
+    Identifier     "Keyboard0"
+    Driver         "kbd"
+EndSection
+
+Section "Monitor"
+    Identifier     "Monitor0"
+    VendorName     "Unknown"
+    ModelName      "Unknown"
+    HorizSync       28.0 - 33.0
+    VertRefresh     43.0 - 72.0
+    Option         "DPMS"
+EndSection
+
+# jrivero: be sure of using always the right BusID
+# can be obtained by: nvidia-xconfig --query-gpu-info
+# The current BusID is the one corresponding to GRID K520
+# at AWS machines of the g2 series.
```

Comment:
It is probably worth creating this config as a template and populating it with variables extracted from the nvidia-xconfig utility.

Reply:
Implemented in 18e06402b4ce2e9bf3dab43619646363e2885ce0